src/commcare_cloud/ansible/roles/formplayer/templates/application.properties.j2

```diff
@@ -15,6 +15,7 @@ sentry.dsn={{ formplayer_sentry_dsn }}
 {% endif %}
 sentry.environment={{ env_monitoring_id }}
 sentry.tags.HQHost={{ host }}
+sentry.release={{ formplayer_release_name }}
```

Comment:
I noticed this gets set to something new and arbitrary when we run `cchq <env> update-config`. Not quite sure what we can do about that, just wanted to note it 🤔

Reply:
That's unfortunate but probably not a big deal.